### ISSUE TITLE
Need multiple postprocessing target folders

### ISSUE THREAD

**Issue by nomadturk** [SickRage/SickRage#3636]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: Master
OS: Debian Jessie

What you did:  I am seperating TV and Anime folders. On show details I'm downloading regular TV shows to /Storage/Media/TV folder whereas I've set /Storage/Media/Anime for all my Anime shows. This is done whilst adding shows to Sickrage

What happened: They've been successfully added. But after downloading the shows, Sickrage put them in /TV folder because that's where postprocessing folder is set for. But it seems I can only set 1 post processing folder and thus the issue arises.

What you expected: Moving Anime to /Storage/Media/Anime and TV to /Storage/Media/TV after they're processed.

**Comment by sharkykh** [comment]
This is called Root Directories. You can set as many as you'd like.
**Config -> General:**
![image](https://cloud.githubusercontent.com/assets/10238474/25107549/c40e026e-23d8-11e7-9b33-30726aace4a0.png)

All you have to do is set the root directory for each of the shows.
You can do this for multiple shows using **Manage -> Mass Update**.

Filter out the all of the anime by entering <kbd>n</kbd> (for anime = no) under the anime column.
![image](https://cloud.githubusercontent.com/assets/10238474/25107606/254f2c06-23d9-11e7-9753-0b44f7b0c9a2.png)

Then select all of the shows using the select box under the "Edit" column header,
and click "Edit selected" at the bottom of the page.

![image](https://cloud.githubusercontent.com/assets/10238474/25107646/5a3aaba2-23d9-11e7-83f7-2f390c629e8a.png)
Here you can set the root directory for all of the regular (non-anime) shows,
which is `/Storage/Media/TV` in your case.

After you change it click done. SickRage will refresh all of the show's data. Depending on how many shows you have that may take some time.

You can either wait for it to finish or continue:
Go back to the **Mass Update** page and filter out the regular show by typing <kbd>y</kbd> under the Anime column.
Same as before, select all of the shows and click on "Edit Selected".
Set the root directory for the anime shows (to `/Storage/Media/Anime`) and click done.

**From this point on,
when you add new shows to SickRage you need to make sure that the show is set to the desired root directory.** 
When adding anime, select `/Storage/Media/Anime`, and when adding regular - `/Storage/Media/TV`.

Good luck.

**Comment by nomadturk** [comment]
@sharkykh 

I did so. I already have two root directories. Default is set to TV but all of my Anime is set to be downloaded to `/Storage/Media/Anime` folder.

But once Sickrage downloaded it, I found out the downloaded episodes in a second  folder in `/Storage/Media/TV` 

I moved the the Season 1 folder from within /TV/AnimeName to /Anime/AnimeName, 
I checked whether if that Anime was set to be downloaded to /TV or /Anime, it was set correctly so no, no misconfiguration here, then I came here and opened an issue.

**Comment by nomadturk** [comment]
Ooops...
I just thought more about it and... It might be related to my naming process though

Checking it I am naming my shows as

`/Storage/Media/TV/%SN/Season %S/%S.N.S%0SE%0E - %EN`

But Anime was not seperated from that...

**Comment by sharkykh** [comment]
You can set a separate naming pattern for Anime, so try that.

I don't think I understand what you're saying.
Generally speaking SickRage uses two types of directories:
1. TV Download dir - where your download client stores completed downloads, so the auto post-processing task knows where to look for new files to process.
2. TV Root dir(s) - where SickRage stores TV shows, and where it moves them during post-processing (per show setting of course).

So you're talking about option 2, correct?

**Comment by sharkykh** [comment]
And just to be clear,
**TV Download dir is not supposed to be any of the TV Root dirs, nor is it recommended to have it as a subdirectory of any of the root dirs.**

**Comment by nomadturk** [comment]
OK,


1- Downloads are done via Transmission to `/Storage/MediaUnorganized/torrentDownloads/TV` folder. Both Anime and TV series reside here.

2- After download is complete, Sickrage is set to move 
TV Series to `/Storage/Media/TV`
Anime to `/Storage/Media/Anime`

This is set via root folders.
![chrome_2017-04-18_02-16-34](https://cloud.githubusercontent.com/assets/10355660/25108301/97f9eaee-23dd-11e7-802c-bd0d425d3cc6.png)

![chrome_2017-04-18_02-16-48](https://cloud.githubusercontent.com/assets/10355660/25108302/9afa76c8-23dd-11e7-9d85-0feb2f6733fd.png)


3- At post processing, it didn't occur to me first that it was related to naming step. I thought it was related to Post processing folder. My bad.

![chrome_2017-04-18_02-17-08](https://cloud.githubusercontent.com/assets/10355660/25108320/bb34445a-23dd-11e7-9b87-6e5a7580aedf.png)

At this step, I think it was related to me NOT checking **custom anime** checkbox here.

Right now, my re-naming is set as

For TV Series `/Depo/Media/TV/%SN/Season %S/%S.N.S%0SE%0E - %EN`
and for Anime '/Depo/Media/Anime/%SN/Season %S/%S.N.S%0SE%0E - %EN`

I think after setting the renaming, my problem won't occur again.

I think what I was expecting was for Sickrage to set it automatically but since I have the full path written on re-naming part, it didn't work out as intended.

**Comment by sharkykh** [comment]
Umm, the naming pattern should be relative to the **show dir** meaning,
**you don't need to put %SN in the naming pattern** as SickRage is already doing this per show.

What I mean is, the base folder for the naming patterns is:
`/Your/Root/Dir/Show Name/`

Just select one of the predefined naming patterns and see that the example doesn't contain root dir, or show name.

So what you need to do is,
**For TV Series** `Season %S/%S.N.S%0SE%0E - %EN`
**and for Anime** `Season %S/%S.N.S%0SE%0E - %EN`

**Comment by sharkykh** [comment]
I'm actually quite surprised that setting didn't cause more of a mess... :laughing: 
But yeah, it seems like the naming pattern was the issue.

Now that I've said that, I realize if you don't name your anime differently, you don't have to enable custom anime naming, but that's your call :)

**Comment by nomadturk** [comment]
I think my re-naming schema is a leftover from back when I was using sickrage in a Synology box. I have ran into some troubles back with Sickrage having seperate Show Info Folder with all the metadata and a folder Processed Shows. That's why I have the absolute path set in it.  

Or it was after I had to manually edit the DB to change all show paths, not quite sure. But it was there for a reason that was meaningful back then :D 

I think it's now safe to change it back. This should be the git of my issue.
